
From: William Lee Irwin III <wli@holomorphy.com>

One of these changes means we need to be able to dereference struct device
in include/asm-ia64/dma-mapping.h.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-ia64/dma-mapping.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN include/asm-ia64/dma-mapping.h~ia64-dma_mapping-fix include/asm-ia64/dma-mapping.h
--- 25/include/asm-ia64/dma-mapping.h~ia64-dma_mapping-fix	Thu Aug  5 14:39:22 2004
+++ 25-akpm/include/asm-ia64/dma-mapping.h	Thu Aug  5 14:39:22 2004
@@ -5,7 +5,8 @@
  * Copyright (C) 2003-2004 Hewlett-Packard Co
  *	David Mosberger-Tang <davidm@hpl.hp.com>
  */
-
+#include <linux/config.h>
+#include <linux/device.h>
 #include <asm/machvec.h>
 
 #define dma_alloc_coherent	platform_dma_alloc_coherent
_
